lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -38,6 +39,10 @@ class Driver < Chef::Provisioning::Driver
 
     attr_reader :aws_config
 
+    # We add the appropriate attributes to the base resources for tagging support
+    Chef::Resource::Machine.include(Chef::Provisioning::AWSDriver::AWSTaggable)
+    Chef::Provider::Machine.additional_machine_option_keys << :aws_tags
```

Comment:
This is where I leverage https://github.com/chef/chef-provisioning/pull/424 to declare that `machine` resources should include their `aws_tags` attributes inside the `machine_options` passed to the driver methods like `allocate_machine`.